pills/04-basics-of-language.xml

```diff
@@ -101,14 +108,9 @@
     </para>
 
     <para>
-      Try to use <literal>/</literal> between two numbers:
-    </para>
-
-    <screen><xi:include href="./04/relative-path.txt" parse="text" /></screen>
-
-    <para>
-      Nix parsed <literal>2/3</literal> as a relative path to the current directory. Expressions will
-      be parsed as paths as long as there's a slash. Therefore to specify the current
+      As demonstrated above, expressions will
+      be parsed as paths as long as there's a slash not followed by a space.
+      Therefore to specify the current
```

Comment:
Can you re-indent the lines around 80 characters?

Reply:
Done